rpi64/debimage-rpi64.yaml

```diff
@@ -0,0 +1,161 @@
+{{- $firmware_version := or .firmware_version "1.20200819" -}}
+{{- $mirror := or .mirror "http://deb.debian.org/debian" -}}
+{{- $suite := or .suite "bullseye" -}}
+{{- $kernel := or .kernel "linux-image-arm64" -}}
+{{- $cmdline := or .cmdline " rootwait ro fsck.mode=auto fsck.repair=yes console=ttyS1,115200" -}}
+{{- $image := or .image "debian-rpi64.img" -}}
+
+architecture: arm64
+
+actions:
+  {{ if ne $firmware_version "debian" }}
+  # Download all needed proprietary stuff first
+  - action: download
+    description: Download latest version of the firmware from GitHub
+    url: http://github.com/raspberrypi/firmware/archive/{{ $firmware_version }}.tar.gz
+    unpack: true     # Unpack downloaded file
+    name: firmware # name for usage directory with unpacked content in other actions
+  {{ end }}
+
+  - action: debootstrap
+    suite: {{ $suite }}
+    components:
+      - main
+      - non-free
+    mirror: {{ $mirror }}
+    variant: minbase
+
+  - action: apt
+    description: Install extra packages
+    packages:
+      - adduser
+      - bluetooth
+      - ca-cacert
```

Comment:
`ca-cacert` feels a bit out of place to me. Why pull in CAcert.org in a basic image as this? (Was the intention possibly to pull in `ca-certificates` ?)

Reply:
right! Thank you for catching this.
Indeed `ca-certificates` are needed to allow to build the image from `http://` repo and have a working ssl on the image.